zowex/Cargo.lock

```diff
@@ -448,7 +448,7 @@ checksum = "712e227841d057c1ee1cd2fb22fa7e5a5461ae8e48fa2ca79ec42cfc1931183f"
 
 [[package]]
 name = "zowe"
-version = "0.9.0"
+version = "0.9.1"
```

Comment:
If there are no user-facing changes to daemon behavior in this PR, perhaps there is no need to publish a new binary in the GitHub releases and we should wait to bump the version until a future PR?

Reply:
That seems reasonable. I will change it back.